cmake/Modules/VolkPython.cmake

```diff
@@ -36,12 +36,16 @@ if(PYTHON_EXECUTABLE)
 else(PYTHON_EXECUTABLE)
 
     #use the built-in find script
-    set(Python_ADDITIONAL_VERSIONS 3.4 3.5 3.6)
-    find_package(PythonInterp 2)
+    set(Python_ADDITIONAL_VERSIONS 3.4 3.5 3.6 3.7 3.8)
+    find_package(PythonInterp 3)
+
+    if(NOT PYTHONINTERP_FOUND)
+        find_package(PythonInterp 2.7)
```

Comment:
Thinking somewhere in here you'd want logic setting a common `PYTHON_INTERPRETER_FOUND` and `PYTHON_EXECUTABLE` ... whichever of Py3 or Py2 was found .... variables that will be common once set regardless of which Py is found and how (old / deprecated or current new, or some future way).